### PR TITLE
Laboratory work #3, Maxim Krivenko - 23FPL2

### PR DESCRIPTION
**Pull request by makskrivenko**

### CODE REVIEW

**Review by artyomtugaryov — CHANGES_REQUESTED** [review]
Nothing to review. Hope I see your amazing code soon!

File: lab_1_classify_by_unigrams/main.py [hunk 1]
```diff
@@ -25,13 +25,11 @@ def tokenize(text: str) -> list[str] | None:
     """
     if not isinstance(text, str):
         return None
-    tokenized_list = []
```
Comment (artyomtugaryov) [inline lab_1_classify_by_unigrams/main.py:28]: ? Revert!

File: lab_3_ann_retriever/settings.json [hunk 2]
```diff
@@ -1,3 +1,3 @@
 {
-    "target_score": 0
+    "target_score": 4
```
Comment (artyomtugaryov) [inline lab_3_ann_retriever/settings.json:2]: Try 6

**Review by artyomtugaryov — CHANGES_REQUESTED** [review]

File: lab_3_ann_retriever/main.py [hunk 3]
```diff
@@ -158,6 +192,8 @@ def build(self) -> bool:
             bool: True if built successfully, False in other case
         """
 
+
```
Comment (artyomtugaryov) [inline lab_3_ann_retriever/main.py:195]: revert

File: lab_3_ann_retriever/start.py [hunk 4]
```diff
@@ -30,9 +31,14 @@ def main() -> None:
...
         text = text_file.read()
-    result = None
+    texts, stop_words = open_files()
+    tokenizer = Tokenizer(stop_words)
+    token_list = tokenizer.tokenize_documents(texts)
+
+    result = token_list
     assert result, "Result is None"
 
+    print(result)
```
Comment (artyomtugaryov) [inline lab_3_ann_retriever/start.py:41]: why?

**Review by artyomtugaryov — CHANGES_REQUESTED** [review]

File: lab_3_ann_retriever/main.py (hunk 3 above)
Comment (makskrivenko) [reply lab_3_ann_retriever/main.py:195]: done

**Pull request closed without merging**